src/MySqlConnector/Core/Row.cs

```diff
@@ -441,6 +441,14 @@ public MySqlGeometry GetMySqlGeometry(int ordinal)
 		throw new InvalidCastException("Can't convert {0} to MySqlGeometry.".FormatInvariant(ResultSet.ColumnDefinitions![ordinal].ColumnType));
 	}
 
+	public MySqlDecimal GetMySqlDecimal(int ordinal)
+	{
+		var value = GetValue(ordinal);
```

Comment:
Will likely need a `GetMySqlDecimalCore` method that TextRow and BinaryRow can override to read the raw bytes as UTF-8 and return a string-based MySqlDecimal. (Or `GetMySqlDecimalAsStringCore` if both row kinds can return UTF8 bytes and the common logic can be centralised here.)